.github/workflows/publish-js-recon.yml

```
@@ -93,4 +97,35 @@ jobs:
           
         env:
           DOCKER_SECRET: ${{ secrets.DOCKER_SECRET }}
+          VERSION: ${{ github.event.release.tag_name }}
+
+
+  publish-ghcr:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: docker/login-action@v2
```

actionlint (1.7.7)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)